ui/lib/core/addon/components/form-field.js

```diff
@@ -54,6 +71,16 @@ export default class FormFieldComponent extends Component {
     this.showInput = !!modelValue;
   }
 
+  get hideLabel() {
+    const { type, options } = this.args.attr;
+    // these take precedent over an attrs default label behavior
+    if (type === 'boolean' || type === 'object' || options.isSectionHeader) {
+      return true;
+    }
+    // falsey values render a <FormFieldLabel>
+    return this.shouldHideLabel[options?.editType];
```

Comment:
we want the default behavior to be  _render the label_, but not all attrs have an `editType`, so if `this.shouldHideLabel[options?.editType]` returns `undefined` a `<FormFieldLabel>` will render

Reply:
What if instead `this.shouldHideLabel` is an array of type names, and then you just check for `this.shouldHideLabel.includes[options?.editType]`?

Reply:
I had originally had it that way. But did this instead as a way to document all the existing types in one place? Angel had added a list to the jsdoc above but when making this getter it seemed strange to write almost all of them again and then keep both lists updated. But if that's unhelpful or unintuitive then I can mimic how it was previously in the `form-field.hbs` template

Reply:
I get the desire to document the types, but I don't think the `shouldHideLabel` map is the place to do that. Probably just in the JSDocs until we make this typescript 😏 Although I like Jordan's suggestion of keeping the types in a separate file, although that wouldn't serve your purposes here